### PR TITLE
699/Help bubble icon

### PR DESCRIPTION
**Pull request by Mdemenko1**
## What (if any) features are you implementing?

This work is for issue [699 ](https://github.com/Gary-Community-Ventures/benefits-calculator/issues/699)

Created the component for help icon button, which accepts help text and visibility prop
Example how it looks on the step 5, "Do you have an income?" question 

https://github.com/Gary-Community-Ventures/benefits-calculator/assets/65931890/9a1448c6-cc4d-4a33-9d27-beef61f87529

### CODE REVIEW

**Comment by sydney-devine** [comment]
Are we specifying what color the help text that pops up will be now? It looked really light in the video, so just want to make sure we default that text to black if we're including that piece int his ticket.

File: src/Components/HelpBubbleIcon/HelpButton.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+const HelpButton = ({
+  className,
+  helpText,
+  helpId,
+  isVisible,
+}: {
+  className?: string;
+  helpText: string;
+  helpId?: string;
```
Comment (CalebPena) [inline src/Components/HelpBubbleIcon/HelpButton.tsx:14]: Why is the `helpId` optional?

File: src/Components/HelpBubbleIcon/HelpButton.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { IconButton } from '@mui/material';
+import { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { HelpBubbleIcon } from './helpBubbleIcon.tsx';
+
+const HelpButton = ({
```
Comment (CalebPena) [inline src/Components/HelpBubbleIcon/HelpButton.tsx:6]: No need to change, but this component can't be used if the parent has `display: flex`.

**Review by CalebPena — CHANGES_REQUESTED** [review]

**Comment by CalebPena** [comment]
@sydney-devine Can we create a ticket to add help bubbles to the headers that need them

**Comment by sydney-devine** [comment]
> @sydney-devine Can we create a ticket to add help bubbles to the headers that need them

Example of where it would be needed in a header?

**Comment by CalebPena** [comment]
> > @sydney-devine Can we create a ticket to add help bubbles to the headers that need them
> 
> Example of where it would be needed in a header?

Sorry for the confusion, I meant add the help bubbles where they are needed.

**Comment by sydney-devine** [comment]
> > > @sydney-devine Can we create a ticket to add help bubbles to the headers that need them
> > 
> > 
> > Example of where it would be needed in a header?
> 
> Sorry for the confusion, I meant add the help bubbles where they are needed.

Ah yes, I need to go back through the app with Kathleen to figure that part out though. So once I have that, I will make ticket.

File: src/Components/HelpBubbleIcon/HelpButton.tsx (hunk 1 above)
Comment (Mdemenko1) [reply src/Components/HelpBubbleIcon/HelpButton.tsx:14]: Was not sure if you always have it, changed it to required

**Comment by Mdemenko1** [comment]
@sydney-devine would you like to leave this one where it is right now?

**Comment by sydney-devine** [comment]
> @sydney-devine would you like to leave this one (help icon) where it is right now?

Yes we can leave that one on the income question. Just want to make sure the text when it unfurls will be in black?

**Comment by Mdemenko1** [comment]
> > @sydney-devine would you like to leave this one (help icon) where it is right now?
> 
> Yes we can leave that one on the income question. Just want to make sure the text when it unfurls will be in black?

yes, I changed color to black

image

**Comment by sydney-devine** [comment]
> > > @sydney-devine would you like to leave this one (help icon) where it is right now?
> > 
> > 
> > Yes we can leave that one on the income question. Just want to make sure the text when it unfurls will be in black?
> 
> yes, I changed color to black
> 
> image

Looks great - TY!

**Review by CalebPena — APPROVED** [review]
Looks good. but we should not merge until we have all of the help bubbles.

**Comment by sydney-devine** [comment]
> Looks good. but we should not merge until we have all of the help bubbles.

We can probably go ahead and merge. I'm not going to have other help bubbles ready to go for this sprint. Will circle back with another ticket to implement the rest of them.

**Pull request merged**